src/Symfony/Component/Serializer/Normalizer/ConstraintViolationListNormalizer.php

```diff
@@ -34,19 +34,29 @@ public function normalize($object, $format = null, array $context = array())
         foreach ($object as $violation) {
             $violations[] = array(
                 'propertyPath' => $violation->getPropertyPath(),
-                'message' => $violation->getMessage(),
-                'code' => $violation->getCode(),
+                'type' => sprintf('urn:uuid:%s', $violation->getCode()),
+                'title' => $violation->getMessage(),
             );
             $propertyPath = $violation->getPropertyPath();
             $prefix = $propertyPath ? sprintf('%s: ', $propertyPath) : '';
             $messages[] = $prefix.$violation->getMessage();
         }
 
-        return array(
-            'title' => isset($context['title']) ? $context['title'] : 'An error occurred',
-            'detail' => $messages ? implode("\n", $messages) : '',
-            'violations' => $violations,
+        $result = array(
+            'type' => $context['type'] ?? 'https://symfony.com/validation-error',
```

Comment:
Just asking: should we make this URL future-proof? Will in the future be other errors? For example: serializer errors. If that's the case, maybe we can change it to `https://symfony.com/error/validation` (and `https://symfony.com/error/serialization`, etc.)

Reply:
Good idea, done.